frontend/views/pages/PayGroup.vue

```diff
@@ -3,157 +3,147 @@ page(
   pageTestName='payGroupPage'
   pageTestHeaderName='payGroupTitle'
 )
-  template(#title='') {{ L('Pay Group') }}
+  template(#title='') {{ L('Payments') }}
 
   template(#sidebar='' v-if='hasPayments')
-    .c-summary-item(
-      v-for='(item, index) in paymentSummary'
-      :key='index'
+    month-overview
+
+  p.p-descritpion.has-text-1(
+    data-test='openIncomeDetailsModal'
+    @click='openModal("IncomeDetails")'
+    v-html='introTitle'
+  )
```

Comment:
Per the updated design from @mmbotelho, there is no text displayed at all when the income details have no been entered yet. Instead we need the standard "Add your income details" card here that exists in other pages (e.g. the same one that's on the Contributions page when no income details are entered).